velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -31,12 +35,17 @@ class HdfsFileSystem::Impl {
     hdfsBuilderSetNameNode(builder, endpoint.host.c_str());
     hdfsBuilderSetNameNodePort(builder, atoi(endpoint.port.data()));
     hdfsClient_ = hdfsBuilderConnect(builder);
+#ifdef VELOX_ENABLE_HDFS3
```

Comment:
Which definition has high priority? Do we need to check this tow definitions conflict?